Dockerfile_amd

```diff
@@ -36,7 +36,7 @@ COPY launcher launcher
 RUN cargo build --release
 
 # Text Generation Inference base image for RoCm
-FROM rocm/dev-ubuntu-22.04:5.7 as base
+FROM rocm/dev-ubuntu-22.04:6.0.2 as base
```

Comment:
Let's update to torch nightly once https://github.com/pytorch/pytorch/pull/124362 is merged and released